packages/kamino-sdk/src/Kamino.ts

```diff
@@ -826,7 +835,13 @@ export class Kamino {
         throw new Error(`Position ${strategy.position.toString()} could not be found.`);
       }
       fetchBalances.push(
-        fetchBalance(strategy, pool as PoolT, position as PositionT, collateralInfos, prices).then((balance) => {
+        fetchBalance(
+          strategy,
+          pool as PoolT,
+          position as PositionT,
+          collateralInfos,
+          prices ? prices[strategy.scopePrices.toBase58()] : undefined
```

Comment:
why do we allow it to be undefined?

Reply:
If not specified we get the scope prices account hardcoded in the config - I guess because it's a private method it doesn't matter too much but it seems to be the pattern atm